src/api/products.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const API_URL = process.env.NEXT_PUBLIC_API_URL;
+
+export const fetchProducts = async () => {
```

Comment:
```suggestion
export const getProducts = async () => {
```
Suggest: fetch는 통신하는 행위에 가까우니 좀 더 직관적으로 이렇게 명명하면 좋을 것 같네요

Reply:
Q: Products vs ProductList

Reply:
[참고자료](https://discord.com/channels/1000691595911376926/1000691596402106370/1339553675714105388)

Reply:
1. 직관적인 함수명,변수명으로 수정 [048796a](https://github.com/house-study/guddnboy-market/pull/3/commits/048796a2d7195ec51824df8eb36d6673e1f1da0d)
2. 저는 Products가 더 자연스럽게 느껴져서 Products로 통일시켰습니다! [d443989](https://github.com/house-study/guddnboy-market/pull/3/commits/d44398989a3521c337393e1d8fe3c5e836e0aefa)